src/modules/notifications/services/notifications.service.ts

```diff
@@ -144,32 +149,46 @@ export class NotificationsService {
 		return affectedRows;
 	}
 
+	/**
+	 * Send notifications and email on new visit request
+	 */
 	public async onCreateVisit(
 		visit: Visit,
 		recipientIds: string[],
 		user: User
 	): Promise<Notification[]> {
-		return await this.createForMultipleRecipients(
-			{
-				title: i18n.t('Er is aan aanvraag om je leeszaal te bezoeken'),
-				description: i18n.t('{{name}} wil je leeszaal bezoeken', {
-					name: user.firstName + ' ' + user.lastName,
-				}),
-				visit_id: visit.id,
-				type: NotificationType.NEW_VISIT_REQUEST,
-				status: NotificationStatus.UNREAD,
-			},
-			recipientIds
-		);
+		const [notifications] = await Promise.all([
+			this.createForMultipleRecipients(
+				{
+					title: i18n.t('Er is aan aanvraag om je leeszaal te bezoeken'),
+					description: i18n.t('{{name}} wil je leeszaal bezoeken', {
+						name: user.firstName + ' ' + user.lastName,
+					}),
+					visit_id: visit.id,
+					type: NotificationType.NEW_VISIT_REQUEST,
+					status: NotificationStatus.UNREAD,
+				},
+				recipientIds
+			),
+			this.campaignMonitorService.send({
+				to: visit.spaceMail,
```

Comment:
this should be the email addresses of the recipientIds that are passed to the function.
But maybe we should fetch those in the controller and pass objects instead:
```
    recipients: {id: string, email: string}[],
```

Reply:
updated